sdk/ml/azure-ai-ml/swagger/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-10-01-preview/registries.json

```diff
@@ -516,6 +516,11 @@
           },
           "x-ms-identifiers": [],
           "x-nullable": true
+        },
+        "managedResourceGroupTags": {
```

Comment:
Is this change going to be made to https://github.com/Azure/azure-rest-api-specs/blob/main/specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-10-01-preview/registries.json ?